compiler/rustc_hir_typeck/src/callee.rs

```diff
@@ -722,12 +731,57 @@ impl<'a, 'tcx> FnCtxt<'a, 'tcx> {
         err.emit()
     }
 
+    /// Enforces that things being called actually are callable
+    #[instrument(skip(self, arguments))]
+    pub(super) fn check_callable(
+        &self,
+        hir_id: hir::HirId,
+        span: Span,
+        value: Ty<'tcx>,
```

Comment:
nit:

```suggestion
        callable_ty: Ty<'tcx>,
```

perhaps?